docs/pages/desktop-access/getting-started.mdx

```diff
@@ -459,6 +459,26 @@ computers and Domain Controllers connected to your domain. Select one and click
 A new tab will open and, after a few seconds, you should be logged in to your
 target Windows host.
 
+## Security Hardening
```

Comment:
Should we make this one of the steps, e.g., `7/7. Harden the Desktop Access service`? I think this makes sense to do if this step is a best practice that applies to all readers.

Reply:
Yeah, thought about it. I was concerned about making this already long and arduous guide even longer, but if you think that's a better approach I'm happy to make the change.

Reply:
Looks like we have six guides with seven steps, and seven is the highest number of steps we have in a guide—another step would make this guide one of our longest, but not _the_ longest 🤷 .

I think either way is okay.

Reply:
Unfortunately AD is ancient, complex, has many weird idiosyncrasies, and is allergic to automation.